### DISCUSSION TITLE
掘金小册分页组件

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2698] [插件说明]
描述
---

> 为掘金小册提供分页功能。

地址
---

[https://simpread.ksria.cn/plugins/details/sCkcmhi9Lb](https://simpread.ksria.cn/plugins/details/sCkcmhi9Lb)

功能
---

> 支持 `→` `←` 翻页

![image](https://user-images.githubusercontent.com/81074/130310385-6cbdb60a-20c9-42a8-a614-36e5f420e332.png)

如何使用
---

1. 安装 [掘金小册的适配站点](https://simpread.ksria.cn/sites/details/jOpeOlEYtf)

![image](https://user-images.githubusercontent.com/81074/130347634-4c40627f-681d-4289-8485-c6b15896abc8.png)

2. 将 `https://juejin.cn/book/*/section/*` 加入到延迟加载列表

![image](https://user-images.githubusercontent.com/81074/130310429-f67b33e9-26cc-4711-a734-03122188e3c3.png)

3. 安装 [此插件](https://simpread.ksria.cn/plugins/details/sCkcmhi9Lb)

更新日志
---

### 1.0.0

- 支持 `→` `←` 翻页

引申
---

> 安装插件或带有插件触发器的插件失效的 [解决方案](https://github.com/Kenshin/simpread/discussions/2342)。

**Comment by xqdd** [comment]
无法使用，翻页后地址栏在变但是内容没有变，退出阅读模式后又恢复正常，应该是翻页后html刷新了但是阅读插件没有及时刷新

**Reply by Kenshin** [reply]
可以使用的。

> 当翻页后会自动获取新的数据，只有一种情况出现你说的问题：当你的网络较慢时，翻页后未正常获取。

所以解决方案：

1. 重新测试（看看是否能稳定复现）
2. 请告知 URL

**Comment by Joker1212** [comment]
还是使用不了完全按照流程走的，自动同步也关闭了，动作里面没有这个插件，页面上也没有翻页按钮

**Reply by Kenshin** [reply]
> 自动同步也关闭了，动作里面没有这个插件

自动同步关闭后，动作里面不可能没有这个插件，除非你把配置文件中的插件相关弄乱了...

解决方案是重置配置文件里面的插件。

另外，这个插件是 2021 年的，不知道是否已适配了最新的页面规则，所以不太建议使用。

***

### 一个提示

如果你使用了同步助手的自动同步，需要使用 https://www.yuque.com/kenshin/simpread/fcw60u 安装插件。
